packages/program-boilerplate/src/trigger.ts

```diff
@@ -180,3 +187,34 @@ function handleProgramValidation(
     code: 200,
   };
 }
+
+function handleProgramVariableSchemaRequest(
+  body: ProgramVariableSchemaRequestBody,
+  program: Program,
+): ProgramTriggerResult {
+  const schema = body.schema;
+  const scheduleKey = body.scheduleKey;
+  const triggerType = body.triggerType;
+  const handleSchemaRequest =
+    program['PROGRAM_TRIGGER_VARIABLES_SCHEMA_REQUEST'];
+  if (!handleSchemaRequest) {
+    return {
+      json: {},
+      code: 204,
+    };
+  } else {
+    const newSchema = handleSchemaRequest(schema, triggerType, scheduleKey);
```

Comment:
You should probably wrap this in a try-catch so that if there is an error in the program it will be logged properly and not crash the instance